File: code_examples/4-Functions/2-Functions_with_a_Parameter/Go/main.go
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
 */

package main

import (
	"fmt"
	"math"
)

func calculateArea(radius int) {
	// this function calculates the area of circle

	// process
	area := math.Pi * radius ** 2

	// output
	fmt.Printf("The area is %d cm²\n", area)
}

func main() {
	// input
	var radius int
	fmt.Print("Enter the radius of a circle (cm): ")
	fmt.Scanln(&radius)
	fmt.Println()

	// call functions
	calculateArea(radius)

	fmt.Println("\nDone.")
}

File: code_examples/4-Functions/2-Functions_with_a_Parameter/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
*/

using System;

/*
 * The Program class
 * Contains all methods for performing basic variable usage
*/
class Program {
    static void calculateArea(int radius) {
        // this function calculates the area of circle

        // process
        int area = Math.Pi * radius ** 2;

        // output
        Console.WriteLine($"The area is {area} cm²");
    }
 
    public static void Main (string[] args) {
        // input
        Console.Write("Enter the radius of a circle (cm): ");
        int radius = int.Parse(Console.ReadLine());
        Console.WriteLine("");

        // call functions
        calculateArea(length, width);

        Console.WriteLine("\nDone.");
    }
}
